### DISCUSSION TITLE
申请证书显示如下错误请问如何解决

### DISCUSSION THREAD

**Discussion by zeno980** [0xJacky/nginx-ui#399] [Q&A]
obtain certificate error: error: one or more domains had a problem: [aaa.bbb.ccc] acme: error: 403 :: urn:ietf:params:acme:error:unauthorized :: [ipv6]: Invalid response from http://aaa.bbb.ccc/.well-known/acme-challenge/pundnjFSxD7K1-oIcC-oBZdVwIjihJR7vTFkP70JkfU: 404

**Comment by 0xJacky** [comment]
配置文件中有 location /.well-know 的规则吗
没有的话，手动添加一下![image](https://github.com/0xJacky/nginx-ui/assets/13096985/48b395cb-380d-4e3e-8b18-d043768517c7)

**Comment by zeno980** [comment]
有这个用户，配置文件也有，请问有没有交流qq群之类的，我进去问下

**Comment by 0xJacky** [comment]
没有噢，你可以删掉这个location配置，再重新添加一次试试

**Comment by 0xJacky** [comment]
再检查一下目录下面有没有.well-known 文件夹，有的话要删掉

**Comment by zeno980** [comment]
试过很多次了，麻烦有空的话看下
![image](https://github.com/0xJacky/nginx-ui/assets/44129442/27ef96ec-b987-4583-8b4e-cdbaa1f43f82)
![image](https://github.com/0xJacky/nginx-ui/assets/44129442/f171c8c8-a597-4a35-9878-a8e1d654519c)
![image](https://github.com/0xJacky/nginx-ui/assets/44129442/0ce151fd-e9d8-4255-9626-18c1348047a9)
![image](https://github.com/0xJacky/nginx-ui/assets/44129442/43c32916-25e3-43c5-984a-435f49e9c58c)
![image](https://github.com/0xJacky/nginx-ui/assets/44129442/a5c767ac-11b4-408d-9939-e1c9eb17cfa9)

**Comment by 0xJacky** [comment]
看出问题来了，你不要改 .well-known 的配置，就用我提供的那个默认反代规则

**Comment by zeno980** [comment]
![image](https://github.com/0xJacky/nginx-ui/assets/44129442/712af25d-9c14-49c3-b94e-39dff91a2d43)
是这样吗，还是报错

**Comment by 0xJacky** [comment]
保存了吗

**Comment by zeno980** [comment]
保存了

**Comment by 0xJacky** [comment]
报错还是提示的404吗，实在不行你改签DNS证书吧

**Comment by zeno980** [comment]
还是403，哎，我试试别的

**Comment by 0xJacky** [comment]
404是反代配置错误，403应该是你请求太多次限流了，晚点再试试

**Comment by zeno980** [comment]
![image](https://github.com/0xJacky/nginx-ui/assets/44129442/66550004-bb22-43ee-8fe2-44bbdfd6ee93)
麻烦问下这是啥原因啊，还有这里为啥是http不是https

**Reply by 0xJacky** [reply]
防火墙有开放 80 端口吗，这个挑战默认就是 http
